ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -90,6 +90,11 @@ func composeFlags() []cli.Flag {
 			Usage:  "Specifies the project name to use. Defaults to the current directory name.",
 			EnvVar: "COMPOSE_PROJECT_NAME",
 		},
+		cli.StringFlag{
+			Name:   command.TaskRoleArnFlag,
+			Usage:  "Specifies the TaskRoleArn to use with the task definition.",
+			EnvVar: "TASK_ROLE_ARN",
```

Comment:
We don't really have implementation on reading from EnvVar here. I would remove this for now and we can come back to this when we have a full solution on how we want to handle ECS TaskDef fields that are not supported by compose.